docker-compose.yml

```diff
@@ -31,11 +31,13 @@ services:
       - FOREMAN_FQDN=foreman.example.com
       - FOREMAN_DOMAIN=example.com
       - FOREMAN_RAILS_CACHE_STORE_TYPE=redis
-      - FOREMAN_RAILS_CACHE_STORE_URLS=redis://redis:6379/0
+      - FOREMAN_RAILS_CACHE_STORE_URLS=redis://redis-cache:6379/0
+      - DYNFLOW_REDIS_URL=redis://redis-tasks:6379/0
```

Comment:
It seems `redis-tasks` got translated to `127.0.0.1` for some reason and it's not connecting to the image.

Reply:
how did you execute it?

Reply:
`docker-compose build` `docker-compose run migration, seed` `docker-compose up` :thinking:

Reply:
It didn't get translated per-se. Sidekiq reliable fetch tried to connect to redis before we could configure sidekiq to use the value from this env var.